backend/src/data/CuisineType.ts

```diff
@@ -0,0 +1,8 @@
+export enum CuisineType {
```

Comment:
I agree -- having these as enums are a lot better and will probably be easier to work with. Good choice :)

Reply:
YUp and one nice thing is that postgres has enum types for columns, so if anyone tries to insert any `cuisinetype`/`discountType`/`promotionCategory` that isn't part of the enum, postgres will complain.